### DISCUSSION TITLE
DNS Setup

### DISCUSSION THREAD

**Discussion by ericsonluciano** [vercel/community#5548] [Help] [answered]
### Summary

I have a domain name from Namecheap, and I point it to my main project from Vercel so vercel now handle my dns manager. And I created a subdomain created Arecord to point from other services (digitalocean) and then set up a mailgun to point to the subdomain as the provider for mailing.

And right now, when i always try to `dig blog.domain.com MX/TXT/CNAME` its always show this: `ns1.vercel-dns.com. hostmaster.nsone.net. 1677965344 43200 7200 1209600 60`  but yesterday it's working fine.

Any idea why it's not connecting to my mailgun?

### Steps to Reproduce

Well i just try to connect to vercel my mailgun dns record to vercel

**Comment by amyegan** [comment, marked as answer]
It sounds like you may have changed the nameservers to point the domain at your Vercel project. In that case, the other DNS records would need to be added in [domain](https://vercel.com/docs/projects/domains#) settings on Vercel. If you used A and CNAME records to point the domain at your Vercel project, then you would need to add the subdomain records in domain settings on Namecheap.

If you share the domain here, someone might be able to recognize exactly what's going wrong with the configuration. If you prefer not to share it, you should check the [troubleshooting domains](https://vercel.com/docs/projects/domains/troubleshooting#) guide for more ideas.

https://vercel.com/docs/projects/domains/troubleshooting
